lib/core/sdam/server.js

```diff
@@ -292,6 +292,16 @@ Object.defineProperty(Server.prototype, 'clusterTime', {
   }
 });
 
+Object.defineProperty(Server.prototype, 'autoEncrypter', {
```

Comment:
since this is going on the prototype, it would probably be better to add this as a `get` on the main class:

```js
class Server extends EventEmitter {
  ...
  get autoEncrypter() {
    // Put your function body here
  }
  ...
}
```

Check out the getters for `description` and `name` for more examples.

Reply:
Made this fix.